### PR TITLE
JNI domain tests

### PR DESCRIPTION
**Pull request by ivnsch**

### CODE REVIEW

File: src/simple_logger.rs [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
 #[cfg(not(test))]
 use chrono::Utc;
 use log::*;
-use std::sync::Once;
+use std::fmt;
+use std::sync::{mpsc::Sender, Once};
 
 static INIT: Once = Once::new();
 
+pub static mut IOS_LOG_SENDER: Option<Sender<CoreLogMessageThreadSafe>> = None;
```
Comment (ivnsch) [inline src/simple_logger.rs:12]: Moved the non-iOS specific log things here. Not sure about `SENDER`. This is still coupled with iOS. For now it's just to make it compile.

File: src/android/jni_domain_tests.rs [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    let muscle_aches_j_value = JValue::from(alert.report.muscle_aches);
+    let loss_smell_or_taste_j_value = JValue::from(alert.report.loss_smell_or_taste);
+    let diarrhea_j_value = JValue::from(alert.report.diarrhea);
+    let runny_nose_j_value = JValue::from(alert.report.runny_nose);
+    let other_j_value = JValue::from(alert.report.other);
+    let no_symptoms_j_value = JValue::from(alert.report.no_symptoms);
+
+    let jni_public_report_obj = env.new_object(
+        jni_public_report_class,
+        "(JJIIZZZZZZZ)V",
```
Comment (duskoo) [inline src/android/jni_domain_tests.rs:151]: @i-schuetz  What is this?

File: src/simple_logger.rs (hunk 1 above)
Comment (duskoo) [reply src/simple_logger.rs:12]: SENDER should be used for both platforms (unless if there is a very good reason not to do so).

Comment (ivnsch) [reply src/simple_logger.rs:12]: Fixed in https://github.com/Co-Epi/app-backend-rust/pull/83

File: src/android/jni_domain_tests.rs (hunk 2 above)
Comment (ivnsch) [reply src/android/jni_domain_tests.rs:151]: The JNI signature for the public report constructor https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/types.html

Comment (ivnsch) [reply src/android/jni_domain_tests.rs:151]: (The Android specific one that uses only primitives, that is)

**Pull request merged**